daffodil-runtime1/src/main/scala/org/apache/daffodil/infoset/SAXInfosetOutputter.scala

```diff
@@ -63,8 +63,7 @@ class SAXInfosetOutputter(xmlReader: DFDL.DaffodilParseXMLReader,
       if (diSimple.hasValue) {
         val text =
           if (diSimple.erd.optPrimType.get.isInstanceOf[NodeInfo.String.Kind]) {
-            val s = remapped(diSimple.dataValueAsString)
-            scala.xml.Utility.escape(s)
+            remapped(diSimple.dataValueAsString)
```

Comment:
Do we need to remove this same escaping for other XML infoset outputters that create objects? I think the Scala, W3C, and JDOM infoset outputters do the same thing. The InfosetInputters probably need to be changed to not unescape things as well?

That doesn't necessarily need to be done in this commit, but we should create a bug to investigate it.

Reply:
I'm not sure.  The only reason I really needed to change it here was that Exificient's library was essentially double escaping the XML entity characters, so clearly they were performing some level of escaping already.  It makes sense to me that in SAX the ContentHandler should receive the raw string values and can then perform whatever escaping it needs to do.

For these other formats I don't know them well enough to say if there are any conventions like this with them.  Worth looking into, but probably outside the scope of this PR.